### ISSUE TITLE
Finding the synthetic utterance for a paraphrase

### ISSUE THREAD

**Issue by harmdevries89** [stanford-oval/genie-toolkit#544]
Hi, 

I've downloaded the [paraphrase data](https://oval.cs.stanford.edu/releases/cikm2020/paraphrase.zip) for the schema2qa paper. I'm wondering if I can recover which synthetic utterance has been paraphrased for each example. The first column in the paraphrase file sometimes refers to a synthetic utterance id (e.g. 4150513_0 for the first paraphrase example in the book domain) but other id's seem to be an index (starting at zero). Can someone explain what's going on in this column and indicate whether it's possible to retrieve the corresponding synthetic utterances?  

Thanks!

**Comment by rayslxu** [comment]
Thank you for your interest in our tool and dataset.
We collected the paraphrase data over the years in multiple rounds while we improving our pipeline. And every time we synthesize a new dataset to paraphrase, the ids will be different. So those IDs are not meaningful and cannot be used to map to synthetic sentences now. 

We did not keep the information of the original synthetic prompt for the paraphrase task. But we do have the raw information available on our mturk account. However, as I said, we have done so many rounds of paraphrasing, with different methods, it's actually quite tricky to get back all of them, so it will take some time. I will keep you updated.
